application-task-manager-ui/src/main/resources/TaskManager/Code/TaskReportResultPage.xml

```diff
@@ -0,0 +1,178 @@
+{{velocity output="false"}}
+## Add the display value of an object field to the specified row.
+#macro(updateRow $colname $row $object)
+    #set($fieldDisplayValue = "$!object.display($colname, 'view')")
+    #if($fieldDisplayValue == '')
+      #set($fieldDisplayValue = $services.localization.render("${request.transprefix}emptyvalue"))
+    #end
+    #set($discard = $row.put($colname, $fieldDisplayValue.replaceFirst($regextool.quote('{{html clean="false" wiki="false"}}'), '').replaceAll("$regextool.quote('{{/html}}')$", '')))
+#end
+## Complete the SQL query so that it filters the UserReference properties.
+#macro(addUserFilter $sqlName $fieldName $isList)
+  #set ($fromSql = "${fromSql}, LargeStringProperty ${sqlName}")
+  #set ($userSql = '')
+  #foreach($user in $request.getParameterValues($fieldName))
+    #if ($foreach.count &gt; 1)
+      #set ($userSql = "${userSql} OR ")
+    #end
+    #if ($isList)
+      #set ($userSql = "$!{userSql}${sqlName}.id.value like CONCAT('%', :${sqlName}${foreach.count}, '%')")
+    #else
+      #set ($userSql = "$!{userSql}${sqlName}.id.value like :${sqlName}${foreach.count}")
+    #end
+    #set ($discard = $whereParams.put("${sqlName}${foreach.count}", $user))
+  #end
+  #set ($whereSql = "$!{whereSql} AND ${sqlName}.id.id=obj.id AND ${sqlName}.id.name='${sqlName}' AND ($userSql)")
+#end
+{{/velocity}}
+{{velocity wiki="false"}}
+## &amp;&amp; "$!{request.outputSyntax}" == 'plain'
+#if($xcontext.action == 'get' )
+  $response.setContentType('application/json')
+  #set ($fromSql = ', StringProperty taskId')
+  #set ($whereParams = {})
+  ## Add Space filter.
+  #if("$!request.space" != '')
+    #foreach($space in $request.getParameterValues('space'))
+      #if ($foreach.count &gt; 1)
+        #set ($spaceSql = "${spaceSql} OR ")
+      #end
+      #set ($spaceSql = "$!{spaceSql}doc.space like :docspace${foreach.count} OR doc.space like CONCAT(:docspace${foreach.count}, '%')")
+      #set ($discard = $whereParams.put("docspace${foreach.count}", $space))
+    #end
+    #set ($whereSql = "AND (${spaceSql})")
+  #end
+  ## Add CreateDate filter.
+  #if("$!request.createDate" != '')
+    #set ($fromSql = "${fromSql}, DateProperty createDate")
+    #set ($whereSql = "$!{whereSql} AND createDate.id.id=obj.id AND createDate.id.name='createDate' AND createDate.id.value &gt;= :createDate")
+    #set ($discard = $whereParams.put('createDate', $datetool.toDate('yyyy/MM/dd', $request.createDate)))
+  #end
+  ## Add assignee filter.
+  #if("$!request.taskAssignee" != '')
+    #addUserFilter('assignee' 'taskAssignee' 1)
+  #end
+  ## Add assignee filter
+  #if("$!request.taskCreator" != '')
+    #addUserFilter('creator' 'taskCreator' 0)
+  #end
+  #gridresultwithfilter_buildQuery("$!request.classname" $request.collist.split(',') $fromSql "AND taskId.id.id=obj.id AND taskId.id.name='id' $!{whereSql}" $whereParams)
+  #set ($sql = "SELECT doc.fullName, taskId.value FROM XWikiDocument doc ${sql}")
+  #set($map = {})
```

Comment:
You're missing the tag cloud information. See ``#gridresult_buildTagCloudJSON``.